debian/outfile

```diff
@@ -0,0 +1 @@
+3.0 (quilt)
```

Comment:
Where does `debian/outfile` come from? Shouldn't that be `debian/source/format` ?